utils/src/main/scala/za/co/absa/enceladus/utils/transformations/ArrayTransformations.scala

```diff
@@ -25,10 +25,30 @@ import org.apache.spark.storage.StorageLevel
 import za.co.absa.enceladus.utils.implicits.OptionImplicits.OptionEnhancements
 import za.co.absa.enceladus.utils.udf.UDFNames
 import za.co.absa.spark.commons.implicits.StructTypeImplicits.StructTypeEnhancementsArrays
+import za.co.absa.spark.commons.sql.functions.col_of_path
+import za.co.absa.spark.commons.utils.SchemaUtils
 
 //TODO refactor to use spark commons and generally better code
```

Comment:
Is this comment still relevant? Spark commons was used and the code looks much better - there are no tmp1, tmp2, res1, res2 etc names.

Reply:
Probably not. Good enough now. 😉